Source/WebCore/Modules/applepay/paymentrequest/ApplePayPaymentHandler.cpp

```diff
@@ -888,7 +888,9 @@ ExceptionOr<void> ApplePayPaymentHandler::retry(PaymentValidationErrors&& valida
     if (errors.isEmpty())
         errors.append(ApplePayError::create(ApplePayErrorCode::Unknown, std::nullopt, nullString()));
 
-    ApplePayPaymentAuthorizationResult authorizationResult { ApplePayPaymentAuthorizationResult::Failure, WTFMove(errors) };
+    ApplePayPaymentAuthorizationResult authorizationResult;
```

Comment:
The downside is that if more fields are added, we won't know that they are uninitialized here. I think that I'd duplicate the whole long line with `ENABLE(APPLE_PAY_PAYMENT_ORDER_DETAILS)` ifdef to protect against that.